fhir-examples/src/main/java/com/ibm/fhir/examples/Index.java

```diff
@@ -0,0 +1,62 @@
+package com.ibm.fhir.examples;
+
+public enum Index {
+    /**
+     * Both R4 spec and IBM generated examples
+     */
+    ALL_JSON("/all-json.txt"),
+
+    /**
+     * Small mix of spec and IBM examples used for unit tests to keep build times short
+     */
+    MINIMAL_JSON("/minimal-json.txt"),
+
+    /**
+     * All R4 spec examples
+     */
+    SPEC_JSON("/spec-json.txt"),
+
+    /**
+     * All IBM generated examples
+     */
+    IBM_JSON("/ibm-json.txt"),
+
+    /**
+     * R4 spec and IBM examples less than 1MB - used for concurrency tests
+     */
+    PERFORMANCE_JSON("/performance-json.txt"),
+
+    /**
+     * Both R4 spec and IBM generated examples
+     */
+    ALL_XML("/all-xml.txt"),
+
+    /**
+     * Small mix of spec and IBM examples used for unit tests to keep build times short
+     */
+    MINIMAL_XML("/minimal-xml.txt"),
+
+    /**
+     * All R4 spec examples
+     */
+    SPEC_XML("/spec-xml.txt"),
+
+    /**
+     * All IBM generated examples
+     */
+    IBM_XML("/ibm-xml.txt");
+
+
```

Comment:
minor, but extra space.

Reply:
i don't see it...

Reply:
line 48/49 are empty lines

Reply:
ah, extra line.  I left it on purpose to visually distinguish the enum values from the rest of the file...I thought I had seen it in one of our other enums somewhere, but definitely not attached to it.  let me know if you really want it squashed, otherwise i'll leave it.